lms/static/sass/elements/_navigation.scss

```diff
@@ -73,4 +76,32 @@
       box-shadow: none;
     }
   }
+
+  // +toggling utilities
+  // --------------------
+  .action-toggle-message {
+    @extend %t-title8;
+    @extend %t-ultrastrong;
+    position: absolute;
+    bottom: 0;
+    @include right(44px);
+    display: inline-block;
+    min-width: ($baseline*5);
+    padding: ($baseline/2) ($baseline*0.75);
+    opacity: 0;
+    background-color: $black;
+    color: $white;
+    text-align: center;
+
+    // STATE: is fleeting/temporary
+    &.is-fleeting {
```

Comment:
NOTE: CSS animation used to temporarily show hiding/showing (FYI @polesye and @jmclaus)